activesupport/lib/active_support/message_encryptor.rb

```diff
@@ -1,6 +1,12 @@
 require 'openssl'
 require 'base64'
 require 'active_support/core_ext/array/extract_options'
+require 'active_support/core_ext/securerandom'
+require 'active_support/core_ext/hash/keys'
+require 'active_support/claims'
+require 'active_support/message_verifier'
+require 'active_support/encryptor/agcm'
```

Comment:
So is it okay for these warnings to show up or is there something I can do about this ? Also, about the structure, I am not sure if we want ```JWE``` class to be under ```MessageEncryptor``` or we could have a separate class for it under ```ActiveSupport``` .